task/04-arrays-tasks.js

```diff
@@ -9,257 +9,266 @@
  *                                                                                           *
  *********************************************************************************************/
 
- 
 /**
  * Returns an index of the specified element in array or -1 if element is not found
- * 
+ *
  * @param {array} arr
  * @param {any} value
  * @return {number}
- * 
+ *
  * @example
- *    ['Ace', 10, true], 10    => 1 
- *    ['Array', 'Number', 'string'], 'Date'    => -1 
+ *    ['Ace', 10, true], 10    => 1
+ *    ['Array', 'Number', 'string'], 'Date'    => -1
  *    [0, 1, 2, 3, 4, 5], 5    => 5
  */
 function findElement(arr, value) {
-   throw new Error('Not implemented');
+  return arr.indexOf(value);
 }
 
 /**
  * Generates an array of odd numbers of the specified length
- * 
+ *
  * @param {number} len
  * @return {array}
- * 
+ *
  * @example
- *    1 => [ 1 ] 
- *    2 => [ 1, 3 ] 
+ *    1 => [ 1 ]
+ *    2 => [ 1, 3 ]
  *    5 => [ 1, 3, 5, 7, 9 ]
  */
 function generateOdds(len) {
-   throw new Error('Not implemented');
+  return (new Array(len)).fill(1).map((item, i) => {
+    return item + i * 2;
+  });
 }
 
-
 /**
  * Returns the doubled array - elements of the specified array are repeated twice using original order
- * 
+ *
  * @param {array} arr
  * @return {array}
- * 
+ *
  * @example
- *    ['Ace', 10, true]  => ['Ace', 10, true,   'Ace', 10, true]  
+ *    ['Ace', 10, true]  => ['Ace', 10, true,   'Ace', 10, true]
  *    [0, 1, 2, 3, 4, 5] => [0, 1, 2, 3, 4, 5,   0, 1, 2, 3, 4, 5]
- *    [] => [] 
+ *    [] => []
  */
 function doubleArray(arr) {
-   throw new Error('Not implemented');
+  return arr.concat(arr);
 }
 
-
 /**
  * Returns an array of positive numbers from the specified array in original order
- * 
+ *
  * @param {array} arr
  * @return {array}
- * 
+ *
  * @example
  *    [ 0, 1, 2, 3, 4, 5 ] => [ 1, 2, 3, 4, 5 ]
  *    [-1, 2, -5, -4, 0] => [ 2 ]
- *    [] => [] 
+ *    [] => []
  */
 function getArrayOfPositives(arr) {
-   throw new Error('Not implemented');
+  return arr.filter(item => item > 0);
```

Comment:
JFYI: use braces in each `for`/`while`/`do ... while`/`if` operator or `callback`. You will make less mistakes.